posts/ja/2022-11-01-letstry-instant-on-intel-mac-en.adoc

```diff
@@ -0,0 +1,280 @@
+[AUDIT   ] CWWKT0017I: Web application removed (default_host): https://de537b960bc9:9443/dev/
+[AUDIT   ] CWWKT0017I: Web application removed (default_host): https://de537b960bc9:9443/health/
+[AUDIT   ] CWWKT0017I: Web application removed (default_host): https://de537b960bc9:9443/metrics/
+[AUDIT   ] CWWKT0017I: Web application removed (default_host): https://de537b960bc9:9443/ibm/api/
+[AUDIT   ] CWWKZ0009I: The application guide-getting-started has stopped successfully.
+[AUDIT   ] CWWKE0036I: The server defaultServer stopped after 2 minutes, 32.806 seconds.  
+----
+
+==== Getting the Checkpoint
+
+Now let's get a checkpoint. Since various privileges are required, we will run with the `--privileged` option. By specifying `applications` for the environment variable `WLP_CHECKPOINT`, Open Liberty can save the checkpoint when the application initialization was completed. 
+
+[source]
+----
+$ podman run --name getting-started-checkpoint-container --privileged --env WLP_CHECKPOINT=applications getting-started
+Performing checkpoint --at=applications
+
+WARNING: Unknown module: jdk.management.agent specified to --add-exports
+WARNING: Unknown module: jdk.attach specified to --add-exports
+Launching defaultServer (Open Liberty 22.0.0.11-beta/wlp-1.0.69.cl221020220912-1100) on Eclipse OpenJ9 VM, version 17.0.5-ea+2 (en_US)
+CWWKE0953W: This version of Open Liberty is an unsupported early release version.
+[AUDIT   ] CWWKE0001I: The server defaultServer has been launched.
+[AUDIT   ] CWWKG0093A: Processing configuration drop-ins resource: /opt/ol/wlp/usr/servers/defaultServer/configDropins/defaults/checkpoint.xml
+[AUDIT   ] CWWKG0093A: Processing configuration drop-ins resource: /opt/ol/wlp/usr/servers/defaultServer/configDropins/defaults/keystore.xml
+[AUDIT   ] CWWKG0093A: Processing configuration drop-ins resource: /opt/ol/wlp/usr/servers/defaultServer/configDropins/defaults/open-default-port.xml
+[AUDIT   ] CWWKZ0058I: Monitoring dropins for applications.
+[AUDIT   ] CWWKT0016I: Web application available (default_host): http://940fd476eccc:9080/ibm/api/
+[AUDIT   ] CWWKT0016I: Web application available (default_host): http://940fd476eccc:9080/health/
+[AUDIT   ] CWWKT0016I: Web application available (default_host): http://940fd476eccc:9080/metrics/
+[AUDIT   ] CWWKT0016I: Web application available (default_host): http://940fd476eccc:9080/dev/
+[AUDIT   ] CWWKZ0001I: Application guide-getting-started started in 1.340 seconds.
+[AUDIT   ] CWWKC0451I: A server checkpoint was requested. When the checkpoint completes, the server stops.
+/opt/ol/wlp/bin/server: line 946:   130 Killed                  "${JAVA_CMD}" "$@" >> "${CHECKPOINT_CONSOLE_LOG}" 2>&1 < /dev/null 
+----
+
+Since the container was started without `--rm`` this time, the container environment after execution remains as it is. In this environment, files that record process information with checkpoint are saved.
```

Comment:
```suggestion
This starts the application container and proceeds to start the application. After the application has started the runtime will perform a checkpoint of the process. When the process state has been saved the container exits. Since the container was started without `--rm` this time, the stopped container remains available for inspection. This stopped container, named `getting-started-checkpoint-container`, contains the checkpoint process state information.
```